aptos-move/block-executor/src/executor.rs

```diff
@@ -941,12 +939,9 @@ where
         base_view: &S,
         start_shared_counter: u32,
         shared_counter: &AtomicU32,
-        shared_commit_state: &ExplicitSyncWrapper<(
-            BlockGasLimitProcessor<T>,
-            Option<Error<E::Error>>,
-        )>,
+        shared_commit_state: &ExplicitSyncWrapper<BlockGasLimitProcessor<T>>,
```

Comment:
Didn't understand this statement. Why are we removing `Option<Error<E::Error>>` here?

Reply:
it's separated out.